backend/src/main/java/com/redhat/cloud/notifications/routers/OrgConfigResource.java

```diff
@@ -61,7 +61,7 @@ static class V2 extends OrgConfigResource {
     @Consumes(APPLICATION_JSON)
     @Transactional
     @RolesAllowed(ConsoleIdentityProvider.RBAC_WRITE_NOTIFICATIONS)
-    @Operation(summary = "Save the daily digest UTC time preference. To cover all time zones conversion to UTC, the accepted minute values are 00, 15, 30 and 45.")
+    @Operation(summary = "Set the daily digest time", description = "Sets the daily digest UTC time. The accepted minute values are 00, 15, 30, and 45. Use this endpoint to set the time that daily emails are sent.")
```

Comment:
```diff
- Use this endpoint to set the time that daily emails are sent.
+ Use this endpoint to set the time in which daily emails are sent.
```

Reply:
Let's compromise with 'when':
Use this endpoint to set the time when daily emails are sent.